src/server.ts

```diff
@@ -19,7 +19,8 @@ import { notablePersonBySlugLoader } from './dataLoaders/notablePerson';
 import { userPhotoUrlLoader } from './dataLoaders/user';
 import { photoUrlLoader } from './dataLoaders/photoUrl';
 
-connection.catch(_ => {
+connection.catch(e => {
+  console.error(e);
```

Comment:
Does it make sense to log these things to Sumo Logic?

Reply:
Yeah I think it makes sense.